conf/routes

```diff
@@ -21,7 +21,7 @@ POST    /api2/atoms/:id/assets          controllers.Api2.addAsset(id)
 
 PUT     /api2/atom/:id/publish          controllers.Api2.publishMediaAtom(id)
 PUT     /api/atom/:id/update-metadata   controllers.Api2.updateMetadata(id)
-POST    /api2/atom/:id/asset-active     controllers.Api2.setActiveAsset(id)
+PUT     /api2/atom/:id/asset-active     controllers.Api2.setActiveAsset(id)
```

Comment:
Yep, though you may want to become familiar with the CDS deploy process beforehand as its not riff-raff - https://sites.google.com/a/guardian.co.uk/gnm-multimedia-knowledgebase/home/how-to-deploy-a-cds-code-update

Once this change goes out, CDS will break as until deployed.